config_iter.go

```diff
@@ -56,6 +56,9 @@ func (ci *ConfigIter) Free() {
 func (ci *ConfigIter) Here() (*string, *string, error) {
 	var err *C.tiledb_error_t
 	var cparam, cvalue *C.char
+	defer C.free(unsafe.Pointer(cparam))
```

Comment:
This is wrong, we don't own these strings its owned by the config iterator in C. We must not free them.